project/Dependencies.scala

```diff
@@ -13,7 +13,7 @@ object WellcomeDependencies {
     val monitoring = defaultVersion
     val storage = defaultVersion
     val elasticsearch = defaultVersion
-    val internalModel = "5021.6fbf97350117e797abb25ab41f1a00d20d7899e5"
+    val internalModel = "4988.884bf55a45d76171db88112142b2ee8dd6e37f46"
```

Comment:
`main` now contains code that depends on the most recent version, so this build will fail.